pkg/parser/sysl.wbnf

```diff
@@ -85,7 +85,7 @@ PATH    -> /{[a-zA-Z0-9._-]+}:"/";
 DIGITS -> [1-9][0-9]*;
 TEXT -> [^(\r?\n)]+;
 TEXT_LINE -> /{\s* ( [^\s:(] [^(\r\n:]* [^\s\r\n(:] )};
-APPLICATION_NAME -> (pkg=@):"." > (app=@):"::" > (NAME | QSTRING)+;
+APPLICATION_NAME -> (pkg=@):"." > (app=@):"::" > (NAME | QSTRING | /{[a-zA-Z_][-\w]*(?:[-\t\_]*\w+)*})+;
```

Comment:
I'm guessing the regexp encompasses `NAME` and `STRING`, so those probably aren't required.

Reply:
the regexp is a more flexible `NAME`. `NAME` can only take alphanumeric while the regexp allows whitespace while still trimming the space.

Reply:
actually just realised `NAME` would always be used first. Will replace `NAME`.